modules/migration-about-configuring-proxies.adoc

```diff
@@ -0,0 +1,124 @@
+Workaround: Use a proxy that supports the SPDY protocol.
+
+In addition to supporting the SPDY protocol, the proxy or firewall also must pass the `Upgrade` HTTP header to the API server. The client uses this header to open a websocket connection with the API server. If the `Upgrade` header is blocked by the proxy or firewall, the migration fail with the error message `Upgrade request required`.
+Workaround: Ensure that the proxy forwards the `Upgrade` header.
+
+== Tuning network policies for migrations
+
+OpenShift supports restricting traffic to or from pods using _NetworkPolicy_ or _EgressFirewalls_ based on the network plugin used by the cluster. If any of the source namespaces involved in a migration use such mechanisms to restrict network traffic to pods, the restrictions might inadvertently stop traffic to Rsync pods during migration.
+
+ Rsync pods running on both the source and the target clusters must connect to each other over an OpenShift Route. Existing _NetworkPolicy_ or _EgressNetworkPolicy_ objects can be configured to automatically exempt Rsync pods from these traffic restrictions.
+
+=== NetworkPolicy configuration
+
+==== Egress traffic from Rsync pods
+
+You can use the unique labels of Rsync pods to allow egress traffic from them if the `NetworkPolicy` configuration in source or destination namespaces blocks this type of traffic. The following policy allows *all* egress traffic from Rsync pods in the namespace:
+
+[source, yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-all-egress-from-rsync-pods
+spec:
+  podSelector:
+    matchLabels:
+      owner: directvolumemigration
+      app: directvolumemigration-rsync-transfer
+  egress:
+  - {}
+  policyTypes:
+  - Egress
+----
+
+==== Ingress traffic to Rsync pods
+
+[source, yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-all-egress-from-rsync-pods
+spec:
+  podSelector:
+    matchLabels:
+      owner: directvolumemigration
+      app: directvolumemigration-rsync-transfer
+  ingress:
+  - {}
+  policyTypes:
+  - Ingress
+----
+
+=== EgressNetworkPolicy configuration
+
+`EgressNetworkPolicy` or _Egress Firewalls_ are OpenShift constructs designed to block egress traffic leaving the cluster.
+
+Unlike `NetworkPolicy`, the Egress Firewall works at a project level in that it applies to all pods in the namespace. Therefore, the unique labels of Rsync pods cannot be used to exempt only Rsync pods from the restrictions. However, you can add the CIDR ranges of the source or target cluster to the _Allow_ rule of the policy so that a direct connection can be setup between two clusters.
+
+Based on which cluster the Egress Firewall is present in, you can add the CIDR range of the other cluster to allow _egress_ traffic between the two:
```

Comment:
Suggestion to remove italics:
```suggestion
Based on which cluster the Egress Firewall is present in, you can add the CIDR range of the other cluster to allow egress traffic between the two:
```
There's a previous instance where "egress" is not italicized, and I'm not sure if it requires emphasis here...if it does though, please feel free to disregard 🙂

Reply:
Fixed.